src/status_im2/subs/general.cljs

```diff
@@ -12,6 +13,17 @@
  (fn [visibility-status-updates [_ public-key]]
    (get visibility-status-updates public-key)))
 
+(re-frame/reg-sub
+ :visibility-status-updates/online?
+ (fn [[_ public-key]]
+   [(re-frame/subscribe [:multiaccount/public-key])
+    (re-frame/subscribe [:visibility-status-updates/visibility-status-update public-key])])
+ (fn [[my-public-key visibility-status-update] [_ public-key]]
+   (let [visibility-status-type (:status-type visibility-status-update)]
+     (or (= public-key my-public-key)
```

Comment:
could you elaborate why for user we don't want to check his status and always show online ? what if he selected offline ?

Reply:
hi @flexsurfer, thank you for reviewing PR. fixed